doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3621,6 +3621,8 @@ See also: Proof. in :ref:`TODO-7.1.4-Proofterm`.
    time the term argument of a tactic has one of its holes not fully
    resolved.
 
+   This command is deprecated. Use typeclasses or tactics-in-terms instead.
+
```

Comment:
Please use the `deprecated` directive: http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated

Reply:
Interesting! I'm going to try using this directive more in #7301.
Right now, it isn't used at all.

Reply:
@herbelin I've added the crossrefs. I'm not too fond of the idea to add an replacement example though. It would make the documentation of that feature bigger than many other ones, while it is virtually unused in the wild. The few potential users have already switched to tactics-in-terms AFAICT.

Reply:
> I'm not too fond of the idea to add an replacement example though. It would make the documentation of that feature bigger than many other ones, while it is virtually unused in the wild. The few potential users have already switched to tactics-in-terms AFAICT.

It stands up. OK with that.